src/ReactiveDomain/ReactiveDomain/FrameFormats/TwoByte128X128.cs

```diff
@@ -9,10 +9,7 @@ namespace ReactiveDomain.FrameFormats
     [StructLayout(LayoutKind.Sequential)]
     public unsafe struct TwoByte128X128Frame
     {
-        public long FrameNumber;
-        public fixed byte FrameId[16];
-        public fixed byte VideoId[16];
-        public double OffsetMilliseconds;
+        public VideoFrameHeader FrameHeader;
```

Comment:
The VideoFrameHeader consists of all the fields in the Image childrens' frames, except the byte pointer:
        public long FrameNumber;
        public fixed byte FrameId[16];
        public fixed byte VideoId[16];
        public double OffsetMilliseconds;
vs
    public unsafe struct VideoFrameHeader
    {
        public long FrameNumber;
        public fixed byte FrameId[16];
        public fixed byte VideoId[16];
        public double OffsetMilliseconds;
    }
Am I missing something here? My impression is that the only impact would in Serialization, where the  inner struct name would show up (VideoFrameHeader.FrameNumber vs FrameNumber)  but we aren't using these child classes yet, so no backward compatibility issues. From what I've read, and from my own experience there shouldn't be a memory impact.